GOT-StudentVersion/GOT-StudentVersion/Controllers/ViewController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  ViewController.swift
+//  GOT-StudentVersion
+//
+//  Created by C4Q  on 11/2/17.
+//  Copyright © 2017 C4Q . All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, UISearchBarDelegate {
+    
+    var Episodes = GOTEpisode.allEpisodes // constant
+    
```

Comment:
You're can also improve the functionality of your code by creating a matrix and assigning it the values of a filtered array of seasons. e.g
 var arrOfSeasons = [[GOTEpisode]]() {
        didSet{
            tableView.reloadData()
        }
    }
    
    private func filterSeason () -> [[GOTEpisode]] {
        arrOfSeasons = [GOTEpisode.allEpisodes.filter{$0.season == 1}, GOTEpisode.allEpisodes.filter{$0.season == 2},GOTEpisode.allEpisodes.filter{$0.season == 3},GOTEpisode.allEpisodes.filter{$0.season == 4},GOTEpisode.allEpisodes.filter{$0.season == 5},GOTEpisode.allEpisodes.filter{$0.season == 6}, GOTEpisode.allEpisodes.filter{$0.season == 7}]
        return arrOfSeasons
    }